### ISSUE TITLE
NTR: X-ray: tarsal bone, supernumerary (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8978]
New term request X-ray: tarsal bone, supernumerary (elthat_130816055858): Additional; extra; accessory  UMLS: 0685460

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000593
- label: Supernumerary tarsal bone
- parents: [Abnormality of the tarsal bones (HP:0001850)](https://hpo.jax.org/app/browse/term/HP:0001850)
- pmid: PMID:9272214
- def: The presence of an additional tarsal bone that is not one of the canonical tarsal bones (cuboid, medial, intermediate, and lateral cuneiform, navicular, talus, calcaneus).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8978
- synonyms: Accessory tarsal bone(NONE)
